xwakes/wit/component.py

```diff
@@ -184,13 +303,16 @@ def __radd__(self, other: Union[int, Component]) -> Component:
         :param other: The left addend of an addition
         :return: The sum of self and other if other is a Component, otherwise just self.
         """
+
         # Checks if the left addend, other, is not a Component
-        if not isinstance(other, Component):
+        if  isinstance(other, int):
```

Comment:
I think here we should replace this test by `if other == 0`. If it's another value, it should raise an error (probably we should test this behaviour, but I think that with just this change that's what will happen).

The docstring should also be changed accordingly.

Reply:
thanks!